oaknational/Oak-Web-Application
# feat(zero-rating): use proxied google fonts service

## Description

- points font request at our google fonts proxy

**NB Feel free to merge while i'm away if approved**

## Issue(s)

Fixes #539 

## How to test

1. Go to {cloud link}
2. Check network tab
3. You should see:

![image](https://user-images.githubusercontent.com/12934669/197251695-4834c92c-65ea-4f2c-93b7-6c1666474226.png)




## Checklist
